components/common/comment/WritableComment.tsx

```diff
@@ -101,10 +118,31 @@ export default function WritableComment({
     }
   };
 
+  const [updatedClassNames, setUpdatedClassNames] = useState({
+    bottomSection: '',
+    buttonContainer: '',
+  });
+
+  useEffect(() => {
+    setUpdatedClassNames({
+      bottomSection: `flex items-end ${
+        isMobile && type === 'pickpickpick'
+          ? 'flex-col gap-[0.8rem]'
+          : 'mt-[1.6rem] justify-between'
+      }`,
+      buttonContainer: `w-full flex gap-[1.6rem] items-center ${
+        isMobile && type === 'pickpickpick' ? 'justify-between' : 'justify-end'
+      }`,
+    });
+  }, [isMobile, type]);
+
```

Comment:
useEffect, useState를 사용하신 이유가 있을까요??
기존에 하던 방식처럼 ClassName 내부에 바로 작성해도 되지 않을까 해서요!
```suggestion
 <div
        className={`flex items-end ${
          isMobile && type === 'pickpickpick'
            ? 'flex-col gap-[0.8rem]'
            : 'mt-[1.6rem] justify-between'
        }`}
      >
        <div className='p2 font-light text-gray4'>
          {textCount}/{MAX_LENGTH}
        </div>
        <div
          className={`w-full flex gap-[1.6rem] items-center ${
            isMobile && type === 'pickpickpick' ? 'justify-between' : 'justify-end'
          }`}
        >
```

Reply:
> WritableComment에서 수정하기 폼이 활성화 될 때 buttonContainer스타일이 isMobile값을 제대로 잡지 못해서 웹버전의 스타일링을 적용하고있어요 ,,, ㅜㅜ

- 요 부분때문에 시도하던 코드였어요!

Reply:
요 부분은 나중에 하기로 함!